code/lib/core-common/src/js-package-manager/JsPackageManager.ts

```diff
@@ -55,7 +55,10 @@ export abstract class JsPackageManager {
     basePath?: string
   ): Promise<PackageJson | null>;
 
-  public abstract getPackageVersion(packageName: string, basePath?: string): Promise<string | null>;
+  async getPackageVersion(packageName: string, basePath = this.cwd): Promise<string | null> {
```

Comment:
@yannbf This function was indeed just used by the `add` command. I looked it up wrongly previously.